supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -495,16 +495,19 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
     * Fire and forget events to the metadata service
     */
   private def tellEventMetadata(eventList: Seq[EventStartTime]): Unit = {
-    // The final event is only used as the book-end for the final pairing so the name is never actually used...
-    val offset = eventList.headOption.map(_.timestamp.getOffset).getOrElse(ZoneOffset.UTC)
-    val now = OffsetDateTime.now.withOffsetSameInstant(offset)
-    val tailedEventList = eventList :+ EventStartTime("unused_name", now)
-    tailedEventList.sliding(2).zipWithIndex foreach {
-      case (Seq(eventCurrent, eventNext), index) =>
-        val eventKey = s"executionEvents[$index]"
-        tellMetadata(s"$eventKey:description", eventCurrent.name)
-        tellTimeMetadata(s"$eventKey:startTime", eventCurrent.timestamp)
-        tellTimeMetadata(s"$eventKey:endTime", eventNext.timestamp)
+    eventList.headOption foreach { firstEvent =>
+      // The final event is only used as the book-end for the final pairing so the name is never actually used...
+      val offset = firstEvent.timestamp.getOffset
+      val now = OffsetDateTime.now.withOffsetSameInstant(offset)
+      val lastEvent = EventStartTime("unused_name", now)
```

Comment:
I just saw that the Queen's jetpacking has been removed in Cromwell v2. Much sadness.